improver_tests/spotdata/spotdata/test_SpotExtraction.py

```diff
@@ -314,6 +320,20 @@ def test_cube_with_leading_dimensions(self):
         self.assertEqual(result.coord("realization"), expected_coord)
         self.assertDictEqual(result.attributes, self.expected_attributes)
 
+    def test_cell_methods(self):
+        """Test cell methods from the gridded input cube are retained on the
+        spotdata cube."""
+        expected_cell_methods = (
+            iris.coords.CellMethod("maximum", coords="time", intervals="1 hour"),
+        )
+        plugin = SpotExtraction(neighbour_selection_method="nearest_land")
+        result = plugin.process(
+            self.neighbour_cube,
+            self.diagnostic_cube_xy,
+            new_title="IMPROVER Spot Forecast",
+        )
+        self.assertEqual(result.cell_methods, expected_cell_methods)
```

Comment:
Make this test against self.cell_methods

Reply:
Updated.